webview/build.gradle.kts

```diff
@@ -102,7 +102,14 @@ fun org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget.setUpiOSObserver()
     }
 }
 
-mavenPublishing {
-    publishToMavenCentral(com.vanniktech.maven.publish.SonatypeHost.S01, automaticRelease = true)
-    signAllPublications()
+publishing {
```

Comment:
## ⚠️ Sensitive Data

This is an absolute **no-go**, if you want to temporarily publish a package to your own repository either use a properties file and exclude it from pushing or at least create a **private** fork where you apply those changes.

https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository